### ISSUE TITLE
Connection terminated with error: -1

### ISSUE THREAD

**Issue by All-StarsPixel** [GaryOderNichts/moonlight-wiiu#36]
I keep getting this error when connecting and trying again freezes the console. My goal is to have 2 screens for 3DS emulation on the Steam Deck. Does anyone know how to fix this error?

Edit: Looking at the Sunshine logs, there are many errors featuring -1 so mabye that's related

Edit 2: At the bottom of the logs, it says Ping Timeout so that might be why its freezing.

**Comment by Martinz64** [comment]
i have this issue too

**Comment by jackgraddon** [comment]
Having a similar issue, my logs don't show much in the way of helping me understand why.
```
[2024:01:25:09:53:28]: Info: CLIENT CONNECTED
[2024:01:25:09:53:28]: Info: Gamepad 0 will be DualShock 4 controller (manual selection)
[2024:01:25:09:53:28]: Info: 
Device Description : AMD Radeon Graphics
Device Vendor ID   : 0x00001002
Device Device ID   : 0x000015BF
Device Video Mem   : 4004 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 5992 MiB
Feature Level      : 0x0000B100
Capture size       : 1280x720
Offset             : 0x0
Virtual Desktop    : 1280x720
[2024:01:25:09:53:28]: Info: Active GPU has HAGS disabled
[2024:01:25:09:53:28]: Info: Using realtime GPU priority
[2024:01:25:09:53:28]: Info: Desktop resolution [1280x720]
[2024:01:25:09:53:28]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2024:01:25:09:53:28]: Info: Display refresh rate [120Hz]
[2024:01:25:09:53:28]: Info: Requested frame rate [60fps]
[2024:01:25:09:53:28]: Info: 
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0.641602,0.322266]
Green Primary      : [0.299805,0.603516]
Blue Primary       : [0.146484,0.0605469]
White Point        : [0.3125,0.329102]
Min Luminance      : 0.5 nits
Max Luminance      : 270 nits
Max Full Luminance : 270 nits
[2024:01:25:09:53:28]: Info: Capture format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2024:01:25:09:53:28]: Info: SDR color coding [Rec. 601]
[2024:01:25:09:53:28]: Info: Color depth: 8-bit
[2024:01:25:09:53:28]: Info: Color range: [MPEG]
[2024:01:25:09:53:46]: Info: CLIENT DISCONNECTED
```
With additional research, I am not sure what is causing the -1 disconnect error. If I reconnect after, the client freezes and becomes unresponsive, forcing me to forcefully power down the Wii U. My host is connected via Ethernet to a router that is near to the console. This error does not persist on other platforms, such as my phone or Apple TV.

**Comment by GaryOderNichts** [comment]
There was an issue where the console froze after reconnecting several times. This has been fixed in [version 1.6](https://github.com/GaryOderNichts/moonlight-wiiu/releases/tag/wiiu-v1.6).

Regarding the errors, make sure to lower your bitrate if your network connection cannot keep up. See the newly added [Troubleshooting section](https://github.com/GaryOderNichts/moonlight-wiiu#troubleshooting).
